py4DSTEM/process/ptychography/ssb.py

```diff
@@ -1,16 +1,225 @@
 import time
-from ...io.datastructure import DataCube
-import sigpy as cp
+from py4DSTEM.io.datastructure import DataCube
+import sigpy as sp
+from sigpy import config
+if config.cupy_enabled:
+    import cupy as cp
 import torch as th
 import numpy as np
+import torch.nn as nn
 from .utils import *
 from py4DSTEM.process.utils import *
 from py4DSTEM.process.utils import plot
 from tqdm import trange
 
+def sector_mask(shape, centre, radius, angle_range=(0,360)):
+    """
+    Return a boolean mask for a circular sector. The start/stop angles in
+    `angle_range` should be given in clockwise order.
```

Comment:
docstring - please add arguments and return value(s)

Reply:
removed